### DISCUSSION TITLE
Workaround for bug #1

### DISCUSSION THREAD

**Discussion by trlkly** [hrj/sloth#20] [Ideas]
I understand that, due to the asynchronous start of tabs and extensions, it is not possible to guarantee that all tabs are unloaded at startup. 

However, why couldn't you just execute an "Unload All Tabs" after the extension loads? That way any tabs that started loading would get discarded immediately? If you don't want this to be the default, you could add it as an option. 

This would also possibly fix another bug I'm going to file.

**Comment by hrj** [comment]
I don't remember if it was already tried and discarded because of errors. Feel free to try and raise a PR if it works.

One note: this action should not trigger on first run after installation, because we don't want to discard running tabs at the time of installation.
